docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/radio/radio.scss

```diff
@@ -5,11 +5,39 @@
 @import '../../00-base/mixins/form';
 
 .civictheme-radio {
+  $root: &;
+
   position: absolute;
+  display: flex;
+  align-items: center;
 
   .civictheme-radio__element {
     @include civictheme-checkbox-radio('radio');
   }
+
+  &.civictheme-theme-light {
+    #{$root}__label {
+      color: $civictheme-radio-label-light-color;
+
+      @include civictheme-typography('text-regular');
```

Comment:
please do not use non-colour mappings in the colo-related specific rules.  Please move this to the outside of `.civictheme-theme-*` rule